src/form/grid/Cell.tsx

```diff
@@ -63,8 +63,8 @@ const Cell = ({ node: el, form }: any) => {
 
   if (
     shouldElementHide({
-      fields: activeStep.servar_fields,
-      element: el
+      element: el,
+      fieldValues
```

Comment:
fieldValues is global and does not need to be passed as a parameter into functions

Reply:
I get your point yesterday @ericpias regarding it being easy to pass in for testing, but we already need to handle it being global for all the other code so it is easiest to be consistent - rather than need to pass in the value or mock depending on what is being tested